packages/react-meteor-accounts/package.js

```diff
@@ -0,0 +1,29 @@
+/* global Package */
+
+Package.describe({
+  name: 'react-accounts',
+  summary: 'React hook for reactively tracking Meteor Accounts data',
+  version: '1.0.0-beta.1',
```

Comment:
Do we want to do a beta.2 release or jump to RC or go straight for a release?

Reply:
We could probably hit RC on this. I'd do one round of RC at least, before going to release.

Reply:
Sounds good! Feel free to change beta to rc and then we need to get @filipenevola to release it.

Reply:
Let's change this to rc.1 and go @filipenevola